tests/acceptance/run.sh

```diff
@@ -670,6 +670,7 @@ fi
 
 # # If we are running in drone CI, then sleep for a bit to (hopefully) let the
 # # drone agent send all the output to the drone server.
+# NOTE: Not using this anymore, please enable it if needed
```

Comment:
In case of not using, what's the point of keeping it?

Reply:
not sure if this is needed or not. but it was commented out months or maybe years ago. haven't noticed any issues but needs a careful look